src/pages/explore/api/use-stats.ts

```diff
@@ -0,0 +1,14 @@
+'use client';
+
+import { useQuery } from '@tanstack/react-query';
+import type { StatsData } from '@/shared/api/server/stats';
+import { innerFetch } from '@/shared/utils/inner-fetch';
+
+export const useStats = () => {
+  return useQuery<StatsData>({
+    queryKey: ['stats'],
```

Comment:
question: does this need a duration window?

Reply:
it is hardcoded in sql, don't think we'll need dynamic window for this